lib/Data/Validate/Sanctions/Redis.pm

```diff
@@ -0,0 +1,230 @@
+package Data::Validate::Sanctions::Redis;
+
+use strict;
+use warnings;
+
+use parent 'Data::Validate::Sanctions';
+
+use Data::Validate::Sanctions::Fetcher;
+use Scalar::Util    qw(blessed);
+use List::Util      qw(max);
+use JSON::MaybeUTF8 qw(encode_json_utf8 decode_json_utf8);
+use Syntax::Keyword::Try;
+
+# VERSION
+
+sub new {
+    my ($class, %args) = @_;
+
+    my $self = {};
+    $self->{redis_read}  = $args{redis_read} or die 'Redis read connection is missing';
```

Comment:
Again. Remove reference of `redis` and pass the config like

```
{
"write_connection":"<redis>",
"read_connection":""
}
```

Reply:
The new code just needs a single `redis` argument. I will rename it simply to `connection` referring to my argument above: https://github.com/binary-com/perl-Data-Validate-Sanctions/pull/45#discussion_r985679851